crates/pop-contracts/src/node/mod.rs

```diff
@@ -36,50 +42,71 @@ pub async fn is_chain_alive(url: url::Url) -> Result<bool, Error> {
 	}
 }
 
-/// Checks if the `substrate-contracts-node` binary exists
-/// or if the binary exists in pop's cache.
-/// returns:
-/// - Some("", <version-output>) if the standalone binary exists
-/// - Some(binary_cache_location, "") if the binary exists in pop's cache
-/// - None if the binary does not exist
-pub fn does_contracts_node_exist(cache: PathBuf) -> Option<(PathBuf, String)> {
-	let cached_location = cache.join(BIN_NAME);
-	let standalone_output = cmd(BIN_NAME, vec!["--version"]).read();
-
-	if standalone_output.is_ok() {
-		Some((PathBuf::new(), standalone_output.unwrap()))
-	} else if cached_location.exists() {
-		Some((cached_location, "".to_string()))
-	} else {
-		None
+/// A supported chain.
+#[derive(Debug, EnumProperty, PartialEq, VariantArray)]
+pub(super) enum SoloChain {
+	/// Parachain containing core Polkadot protocol features.
```

Comment:
```suggestion
	/// Minimal Substrate node configured for smart contracts via pallet-contracts.
```

Use the repo description.